pkg/cmd/openshift-tests/disruption/watch-endpointslice/endpointslice_controller.go

```diff
@@ -8,32 +8,37 @@ import (
 	"sync"
 	"time"
 
-	"github.com/openshift/origin/pkg/monitor/monitorapi"
-
-	discoveryv1 "k8s.io/api/discovery/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 
 	"github.com/openshift/origin/pkg/monitor/backenddisruption"
-
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	discoveryv1 "k8s.io/api/discovery/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	coreinformers "k8s.io/client-go/informers/core/v1"
 	discoveryinformers "k8s.io/client-go/informers/discovery/v1"
+	corelisters "k8s.io/client-go/listers/core/v1"
 	discoverylisters "k8s.io/client-go/listers/discovery/v1"
-
-	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog/v2"
 	"k8s.io/kubernetes/pkg/controller"
 )
 
 type EndpointSliceController struct {
-	backendPrefix string
-	serviceName   string
-	recorder      monitorapi.RecorderWriter
-	outFile       io.Writer
-
-	endpointSliceLister  discoverylisters.EndpointSliceLister
-	endpointSlicesSynced cache.InformerSynced
+	backendPrefix     string
+	namespaceName     string
+	serviceName       string
+	myNodeName        string
+	stopConfigMapName string
```

Comment:
Godoc plz, the CLI flags covers it pretty good but that's a ways from here, would be good to have it on the final stopping point. Clarify that in normal operation the configmap is created from the origin monitor test.